### PR TITLE
Mobile contact form

### PR DESCRIPTION
**Pull request by amrendrakind**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,231 +1,301 @@
...
+            placeholder="Your name"
+            required
+          />
+          <input
+            type="email"
+            name="mail"
+            value="natajayanj14@gmail.com"
+            placeholder="Your email"
+            required
+          />
```
Comment (ReshyResh) [inline index.html:287]: - [ ] Please remove the hardcoded values like `Yeremias` so the user will only see the placeholder and will fill his own informations.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @amrendrakind , 

### Good job so far!
  - ✔ Form works and it sends data through Formspree.
  - ✔ Correct use of Flexbox to lay out the elements.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ibnballo1) [reply index.html:287]: Thanks @ReshyResh , but it is there in the figma design... should I just remove the placeholder instead

Comment (ReshyResh) [reply index.html:287]: The hard coded value is different from the placeholder. The way it is right now, the user will have to delete the data that is already on the field in order to input his own. With only a placeholder, as soon as the user starts typing the input, the placeholder text will go away. The text on Figma is there just to demonstrate to you that it's a form 👍

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @Ibnballo1 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ibnballo1) [reply index.html:287]: Yeah, i understood that but i thought that was how it was in the figma design

[the remark this answers is not in the document]
Comment (Ibnballo1) [reply index.html:287]: Thanks @thecodechaser , should i remove that of the email too

File: index.html [hunk 2]
```diff
@@ -1,231 +1,301 @@
...
+          <textarea
+            maxlength="500"
+            placeholder="write your message here"
+            required
+          ></textarea>
+          <button type="submit">Get In Touch</button>
+        </form>
```
[the remark this answers is not in the document]
Comment (Ibnballo1) [reply index.html:294]: Thanks @thecodechaser , i forgot to remove the border from the background image but will correct that now

**Review by thecodechaser — APPROVED** [review]
## Hi @Ibnballo1 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**